diff --git a/node_modules/@sveltejs/vite-plugin-svelte/dist/index.js b/node_modules/@sveltejs/vite-plugin-svelte/dist/index.js
index cf5ddfa..2a6b363 100644
--- a/node_modules/@sveltejs/vite-plugin-svelte/dist/index.js
+++ b/node_modules/@sveltejs/vite-plugin-svelte/dist/index.js
@@ -377,8 +377,8 @@ function parseToSvelteRequest(id, filename, rawQuery, root, timestamp, ssr) {
   if (query.url || query.raw) {
     return;
   }
-  const normalizedFilename = normalize(filename, root);
-  const cssId = createVirtualImportId(filename, root, "style");
+  const normalizedFilename = normalize(filename, root, query.page);
+  const cssId = createVirtualImportId(filename, root, "style", query.page);
   return {
     id,
     filename,
@@ -389,8 +389,11 @@ function parseToSvelteRequest(id, filename, rawQuery, root, timestamp, ssr) {
     ssr
   };
 }
-function createVirtualImportId(filename, root, type) {
+function createVirtualImportId(filename, root, type, page) {
   const parts = ["svelte", `type=${type}`];
+  if (page) {
+    parts.push(`page=${page}`);
+  }
   if (type === "style") {
     parts.push("lang.css");
   }
@@ -410,8 +413,8 @@ function parseRequestQuery(rawQuery) {
   }
   return query;
 }
-function normalize(filename, normalizedRoot) {
-  return stripRoot(normalizePath(filename), normalizedRoot);
+function normalize(filename, normalizedRoot, page) {
+  return stripRoot(normalizePath(filename), normalizedRoot) + (page ? `?${page}` : '');
 }
 function existsInRoot(filename, root) {
   if (filename.startsWith(VITE_FS_PREFIX)) {
